OpenDreamRuntime/DreamManager.cs

```diff
@@ -39,16 +39,17 @@ partial class DreamManager : IDreamManager {
         public List<DreamObject> Datums { get; set; } = new();
         public Random Random { get; set; } = new();
 
-        public void Initialize() {
+        //TODO This arg is awful and temporary until RT supports cvar overrides in unit tests
+        public void Initialize(string? testingJson) {
```

Comment:
This is more than just the testing json now, and shouldn't be null.
```suggestion
        public void Initialize(string jsonPath) {
```